Madness/Map.swift

```diff
@@ -24,3 +42,11 @@ infix operator >>- {
 	associativity left
 	precedence 150
 }
+
+/// Map operator.
+infix operator <^> {
+	associativity left
+
+	// Higher precedence than `>>-`.
+	precedence 155
```

Comment:
One potential complication is that Runes doesn’t give a precedence for this operator, meaning it defaults to `100`.

I think if a file imports both Runes and Madness they can select which precedence they desire by explicitly redeclaring the operator, so I think that’s ok.